server/lib/notifications/agents/agent.ts

```diff
@@ -27,3 +27,9 @@ export interface NotificationAgent {
   shouldSend(type: Notification, payload: NotificationPayload): boolean;
   send(type: Notification, payload: NotificationPayload): Promise<boolean>;
 }
+
+export const userNotificationTypes: Notification[] = [
+  Notification.MEDIA_APPROVED,
+  Notification.MEDIA_DECLINED,
+  Notification.MEDIA_AVAILABLE,
+];
```

Comment:
What about adding this as a static method/value in the `BaseAgent` since we extend that for every agent anyways? Then you don't need to import this into the other files.

Reply:
That definitely sounds better!  Will fix shortly